src/libraries/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -151,7 +152,7 @@ internal static SafeSslHandle AllocateSslContext(SslProtocols protocols, SafeX50
 
                 if (hasCertificateAndKey)
                 {
-                    SetSslCertificate(innerContext, certHandle, certKeyHandle);
+                    SetSslCertificate(innerContext, certHandle!, certKeyHandle!);
```

Comment:
I'm surprised Roslyn can't figure this out themselves. We check `!= null` above and store it in a bool.

Reply:
https://github.com/dotnet/roslyn/issues/34665
https://github.com/dotnet/csharplang/issues/2388

Reply:
I presume that's intentional though. There's only so many scenaria you can support  before complexity goes out of hand.